packages/backup-restore-operator/charts/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: backup-restore-operator-installer
+subjects:
+- kind: ServiceAccount
+  name: backup-restore-operator-serviceaccount
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
It shouldn't bind to cluster-admin instead using specific rules that your controller uses

Reply:
@StrongMonkey this chart needs to be able to modify any resouces. Hence the cluster-admin role was decided in the design meeting